chicago/bills.py

```diff
@@ -24,7 +24,8 @@ class ChicagoBillScraper(LegistarAPIBillScraper, Scraper):
     VOTE_OPTIONS = {'yea' : 'yes',
                     'rising vote' : 'yes',
                     'nay' : 'no',
-                    'recused' : 'excused'}
+                    'recused' : 'excused',
+                    None: 'abstain'}
```

Comment:
How do we know this surprising fact is true?  Please add a code comment that explains this.

Reply:
we don't; this is the pattern we used in metro, but it may well be different for chicago. 

specifically, there's a comment here suggesting we don't capture voice votes:

https://github.com/opencivicdata/scrapers-us-municipal/blob/26aa22f7804e9ee28db90b6e3456518a6d747cb5/chicago/bills.py#L101-L102

all three of the events with null votes in the last two weeks were voice votes:

- [histories](http://webapi.legistar.com/v1/chicago/matters/101988/histories) / [votes](http://webapi.legistar.com/v1/chicago/eventitems/406835/votes)
- [histories](http://webapi.legistar.com/v1/chicago/matters/87964/histories) / [votes](http://webapi.legistar.com/v1/chicago/eventitems/361658/votes)
- [histories](http://webapi.legistar.com/v1/chicago/matters/21350/histories) / [votes](http://webapi.legistar.com/v1/chicago/eventitems/137691/votes)

so, with that context, abstention seems wrong, but then there's an outstanding question of how/whether to capture voice votes. do you have thoughts on this?

Reply:
some more context would be helpful. was the behavior previously that we captured that a voice vote happened, but didn't capture individual votes, because they weren't added to legistar?

Reply:
we're going to capture that a voice vote happened, but not capture the individual null votes.